classes/teacherclass.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+# teacher subclass
+
+class Teacher < Person
```

Comment:
- [ ] Kindly remove the suffix class from the class files to ensure a proper class-file naming convention. This will prevent import errors. Kindly change the file `teacherclass.rb` to `teacher.rb`